### DISCUSSION TITLE
Accidentally ran it on Windows 10, explorer no longer starting

### DISCUSSION THREAD

**Discussion by itsTyrion** [valinet/ExplorerPatcher#1293] [General]
I accidentally ran the setup on my real system with Windows 10 instead of the Windows 11 VM, explorer no longer starts, relogging doesn't help, had to open my browser via Task-Manager -> File -> New Task...

What do I have to remove/change and how? Command line and regedit usage, if needed, is not a problem for me.

**Comment by pyrates999** [comment]
Follow the instructions here: https://github.com/valinet/ExplorerPatcher/wiki#uninstalling

**Comment by itsTyrion** [comment]
Forget to write the result: I chose to rename the setup (powershell), which worked. 
Is it expected that Windows 10 Explorer just straight up dies? IF so, maybe there should be a check for the Windows version?
